packages/app-store/stripepayment/lib/PaymentService.ts

```diff
@@ -89,7 +89,7 @@ export class PaymentService implements IAbstractPaymentService {
             stripe_publishable_key: this.credentials.stripe_publishable_key,
             stripeAccount: this.credentials.stripe_user_id,
           }) as unknown as Prisma.InputJsonValue,
-          fee: paymentFee,
+          fee: 0,
```

Comment:
Assuming this is intentional BTW

Reply:
Will keep it out of this PR for now. But basically we might want to do this to support multiple countries.